### DISCUSSION TITLE
My first HTML Project on my Software Journey

### DISCUSSION THREAD

**Discussion by shakscode19** [community/community#129562] [New to GitHub]
Hello folks,
I just wrote my first lines of HTML code here https://github.com/shakscode19/week-1-html-setup-shakscode19.git
What do you think? Anything I can do to make it cleaner and more readable?

**Comment by manuel14mds** [comment]
very good 👍

**Reply by shakscode19** [reply]
Thanks Manuel14mds

**Comment by Sudhaanshuu** [comment]
I would love to know if you would be willing to share the problems you faced while working on your first project, as well as how you overcame them.

**Reply by shakscode19** [reply]
I am enrolled in a self-paced course by Power Learn Project Kenya. I went through the online videos and after creating the html file, I checked with chatgpt and realized that I could make my code shorter by removing the 'body' tags I had created and using a single 'body' tag for the 3 or 4 headings I was creating. I had a bit of a challenge understanding Forms, specifically the attributes, so I checked stack overflow and W3 schools for suggestions.

**Reply by Sudhaanshuu** [reply]
That's nyc. keep coding, Keep growing.

**Reply by shakscode19** [reply]
thanks

**Comment by romhenri** [comment]
Hey, man.

I would recommend make a `head` like this:

```html
<head>
    <meta charset="UTF-8" />
    <link rel="icon" type="image/svg+xml" href="/icon.svg" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>My Page</title>
</head>
  ```
  
  To you use charset, icon, responsiveness rule and the title.
  
  Good lucky and if you need, contact me.

**Reply by shakscode19** [reply]
Thank you very much for the recommendation. I'm going to incorporate it asap.
